app/models/reporting/decisions.rb

```diff
@@ -160,6 +160,14 @@ class Reporting::Decisions < ApplicationRecord
       merge(contacts)
   end
 
+  scope :contacts_in_order, ->(contact_array, order) do
+    contacts = ClientOpportunityMatchContact.where(contact_order: order)
+    contacts.where(contact_id: contact_array) unless contact_array.blank?
+
+    joins(match: :contacts).
+      merge(contacts)
+  end
```

Comment:
Should this return an ordered scope?  The name is a bit odd.  I think I get what's going on, but maybe `contacts_with_order` would be more clear.

Reply:
makes sense to me, I'll update the name to make it more clear.